onnxruntime/core/common/str_helper.cc

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#ifdef _WIN32
+#include "core/common/common.h"
+#include <Windows.h>
+#include <assert.h>
+
+namespace onnxruntime {
+std::string ToMBString(const std::wstring& s) {
+  if (s.size() >= std::numeric_limits<int>::max()) throw std::runtime_error("length overflow");
+  const int src_len = static_cast<int>(s.size() + 1);
+  const int len = WideCharToMultiByte(CP_ACP, 0, s.data(), src_len, nullptr, 0, nullptr, nullptr);
+  assert(len > 0);
```

Comment:
>assert(len > 0); [](start = 2, length = 16)

Would this break if the input string was empty?

Reply:
No.

Reply:
Please see the next line.

Reply:
Sorry - I wasn't clear. I meant 'should this fail if the input string is empty'? Or should it just return an empty string in that case?

---
In reply to: [261362978](https://github.com/Microsoft/onnxruntime/pull/520#discussion_r261362978) [](ancestors = 261362978)

Reply:
I think returning an empty string is ok, it will make this function easier to use. Caller don't have to check if the input string is empty before calling this function.

---
In reply to: [261412635](https://github.com/Microsoft/onnxruntime/pull/520#discussion_r261412635) [](ancestors = 261412635,261362978)